src/khoj/processor/tools/run_code.py

```diff
@@ -79,13 +90,12 @@ async def generate_python_code(
     context: str,
     location_data: LocationData,
     user: KhojUser,
-    query_images: List[str] = None,
+    query_images: list[str] = None,
     agent: Agent = None,
     tracer: dict = {},
-) -> List[str]:
+) -> tuple[str, list[str], list[str]]:
```

Comment:
Tuple strikes again 😅. You already know my feedback here.

Reply:
Fixed!